source/unit/Constraints.ooc

```diff
@@ -131,6 +139,10 @@ EqualModifier: class extends Modifier {
 		f := func (value, c: Object) -> Bool { value as Cell<Double> get() == c as Cell<Double> get() }
 		CompareWithinConstraint new(this, Cell<Double> new(correct), f)
 	}
+	to: func ~ldouble (correct: LDouble) -> CompareWithinConstraint {
+		f := func (value, c: Object) -> Bool { value as Cell<LDouble> get() == c as Cell<LDouble> get() }
+		CompareWithinConstraint new(this, Cell<LDouble> new(correct), f)
+	}
```

Comment:
I think you know ooc a lot better than I do, I don't even know if function templates exist in ooc (I know class templates are available).
I don't know, can we have a template class with static `to` method, something like
```ooc
CompareCreator: class <T> {
    to: static func (value: T)-> CompareWithinConstraint {
    to: func ~ulong (correct: ULong) -> CompareConstraint {
		f := func (value, c: Object) -> Bool { value as Cell<T> get() == c as Cell<T> get() }
		CompareConstraint new( ???? , Cell<T> new(correct), f, ComparisonType Equal)
	}
    }
}
```
but what should be in place of `this` then ? I can look into it later, though.